### PR TITLE
Add note to README regarding the state of Marathon

### PR DESCRIPTION
**Pull request by timcharper**

### CODE REVIEW

**Review by tillt — APPROVED** [review]
Thanks Tim!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# Project Status
+
+D2iQ (formerly Mesosphere), that created and sponsored this project from the very beginning, has announced on October 30th, 2020:
+
+> As of [Oct 30, 2020], we are winding down the Mesosphere DC/OS Platform. We will continue to provide support for our current DC/OS customers, but will no longer be investing in new features or capabilities. We continue to work closely with existing customers to migrate […]”
+
+The future of this project is, for now, uncertain. Please consider the above announcement when evaluating if DC/OS is the right solution for future deployments.
```
Comment (Colin1841) [inline README.md:7]:
```suggestion
With the future of this project uncertain, please consider this when evaluating if DC/OS is the right solution for future deployments.
```

**Comment by cmastr** [comment]
@timcharper I think we want to use this content instead:

On October 20, 2020, D2iQ (formerly Mesosphere) announced its intent to discontinue the DC/OS platform.

Based on the changing market needs and very clear feedback from our customers, D2iQ is making the decision to focus our resources on the future, that being our D2iQ Kubernetes Platform (DKP) portfolio of products. To accelerate the delivery of the innovation our customers have become accustomed to from D2iQ, it is necessary for us to align our business to support that objective. We are working closely with existing customers to migrate to the D2iQ Kubernetes Platform and provide them with the enhanced innovation and enterprise capabilities available from Kubernetes. 

Support for DC/OS ends on October 31, 2021. We will continue to provide support for our current DC/OS customers per their contracts, of course. However, we will no longer be investing in new features or capabilities or maintaining the related repositories. Continued use of the DC/OS open source platform, Marathon, and other DC/OS components will be at your own discretion and risk. 
We want to thank all of our loyal customers, particularly those DC/OS users who were fellow pioneers in the growth of the cloud native landscape from the beginning.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# Project Status
+
+Support for DC/OS ends on October 31, 2021. D2iQ will continue to provide support for our current DC/OS customers per their contracts, of course. However, we will no longer be investing in new features or capabilities or maintaining the related repositories. If a customer wishes to continue use of the DC/OS Enterprise platform or other non-free DC/OS components, the customer can purchase an End-of-Life License or Perpetual Use License, however support is not included in these licenses and continued use of DC/OS will be at your own discretion and risk.
+
+We want to thank all of our loyal customers, particularly those DC/OS users who were fellow pioneers in the growth of the cloud native landscape from the beginning.
```
Comment (tillt) [inline README.md:5]:
```suggestion
On October 20, 2020, D2iQ (formerly Mesosphere) announced its intent to discontinue the DC/OS platform.

Based on the changing market needs and very clear feedback from our customers, D2iQ is making the decision to focus our resources on the future, that being our D2iQ Kubernetes Platform (DKP) portfolio of products. To accelerate the delivery of the innovation our customers have become accustomed to from D2iQ, it is necessary for us to align our business to support that objective. We are working closely with existing customers to migrate to the D2iQ Kubernetes Platform and provide them with the enhanced innovation and enterprise capabilities available from Kubernetes.

Support for DC/OS ends on October 31, 2021. We will continue to provide support for our current DC/OS customers per their contracts, of course. However, we will no longer be investing in new features or capabilities or maintaining the related repositories. Continued use of the DC/OS open source platform, Marathon, and other DC/OS components will be at your own discretion and risk.
We want to thank all of our loyal customers, particularly those DC/OS users who were fellow pioneers in the growth of the cloud native landscape from the beginning.
```

**Review by tillt — COMMENTED** [review]
This is @cmastr edit in a suggestion....

**Review by cmastr — APPROVED** [review]

**Review by meichstedt — APPROVED** [review]
Goodbye, Marathon! 👋 

– Tim, let's port it onto USI already ;-)

**Comment by cmastr** [comment]
Approved words:

Support for DC/OS ends on October 31, 2021. We will continue to provide support for our current DC/OS customers per their contracts, of course. However, we will no longer be investing in new features or capabilities or maintaining the related repositories. If a customer wishes to continue use of the DC/OS Enterprise platform or other non-free DC/OS components, the customer can purchase an End-of-Life License or Perpetual Use License, however support is not included in these licenses and continued use of DC/OS will be at your own discretion and risk.
We want to thank all of our loyal customers, particularly those DC/OS users who were fellow pioneers in the growth of the cloud native landscape from the beginning.

**Review by tillt — APPROVED** [review]

**Review by cgawron1 — APPROVED** [review]

**Comment by cmastr** [comment]
Still needs a bit of updating:

The new words are there, but so are the old one. Take out paragraphs 1 and 2, leave 3 and 4

**Comment by timcharper** [comment]
Done

**Comment by jeschkies** [comment]
:cry: bye bye

**Comment by zen-dog** [comment]
Goodbye, Marathon and thanks for everything! 😞

**Pull request merged**